docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -34,72 +47,58 @@ To build an APB:
 
-While signed into Docker Hub from a browser:
+....
+apb push -o
+....
 
-* Create > Create Automated Build
-* Choose Github and allow access to repositories in the aerogearcatalog Github org
-* Look for the repo in the list and click it
-* Make sure the Repository Namespace & Name are correct e.g. aerogearcatalog/metrics-apb. Visibility public is fine.
-* After creating, go to Build Settings
-* Configure 'master' branch to build
-* Configure tags that match a name of `/^[0-9.]+/` to build
-* Save changes and Trigger a build for `master`. If successful, there should be an image with the 'latest' tag.
+Afterwards your APB is ready to be used from the _Service Catalog_.
 
-=== Pushing to Dockerhub
+[NOTE]
+====
+* When pushing an APB it will not be available immediately. 
 
-If you want to push the APB to your own Dockerhub account:
+* Using `make build_and_push` executes all phases at once.
+====
 
-....
-make DOCKERORG="my_org" docker_push
-....
+=== Configure OpenShift Ansible Broker to use Dockerhub Regsitry
 
-NOTE: Set `DOCKERHOST` to override the dockerhost default, which is `docker.io`.
+By default, the installer in this repo configures the OpenShift Ansible Broker to list images in the Service Catalog from OpenShift's local docker registry **and** from the `aerogear` organisation in Dockerhub.
 
-=== Pushing APBs to OpenShift Ansible Broker (OAB) locally
+You can configure the OpenShift Ansible Broker to list images from your Dockerhub account as follows:
 
-. Create a MCP cluster pointing at *your* own docker organisation:
-+
 ....
-ansible-playbook playbook.yml -e "dockerhub_username=<dockerusername>" -e "dockerhub_password=<dockerpassword>" -e "dockerhub_org=<USE_THIS_VALUE>" -e "apb_sync=true" --ask-become-pass
+oc project ansible-service-broker
+oc edit configmap broker-config
 ....
-+
-[NOTE]
-====
-* This copies all the existing APBs in aerogear to the `dockerhub_org` registry.
-
-* Setting `apb_sync=true` pushs all the aerogear catalog to your dockerhub account.
 
-* Developers can use aerogearcatalog organization and openshift local registry for development.
-====
+This will allow you to edit the OAB config in your terminal. Under the `registries` list, add another item:
 
-. Make the APB:
-+
 ....
-make apb_build
+- type: dockerhub
+  name: <some name>
+  org: <your dockerhub username>
+  user: <your dockerhub username>
+  pass: <your dockerhub password>
+  white_list:
+    - ".*-apb$"
 ....
 
-. Push your ABP to the running ASB of your MCP project:
-+
+You can also edit the `broker-config` Config Map in the OpenShift web console under the `ansible-service-broker` project. Check the link:https://github.com/openshift/ansible-service-broker/blob/master/docs/config.md[Ansible Service Broker configuration docs] for more examples.
+
+=== Push to Dockerhub Account
+You can build and push the _APB_ to your own Dockerhub account with the following:
```

Comment:
Should we have some details about why or when you'd do this?
Maybe some caveats to doing this, and how you could `apb relist` to get new/updated images to show up in the catalog?

Reply:
Good point @david-martin To be honest, I'm not sure why you would do this when the local option is better. I considered removing docs around Dockerhub but I figured it might be valuable to show how to reconfigure the broker for different registries.

I'll include a note about `apb relist`

Reply:
Hmm, the only reason for pushing to your own dockerhub that i can see is that it survives a `make clean` (your local registry will be wiped). But then again you can always re-push your apbs.